components/layout_2020/flow/mod.rs

```diff
@@ -1086,30 +1089,32 @@ impl NonReplacedFormattingContext {
         // Margins can never collapse into independent formatting contexts.
         sequential_layout_state.collapse_margins();
         sequential_layout_state.advance_block_position(
-            (pbm.padding_border_sums.block +
-                content_size.block +
-                clearance.unwrap_or_else(Length::zero))
-            .into(),
+            pbm.padding_border_sums.block +
+                content_size.block.into() +
+                clearance.unwrap_or_else(Length::zero).into(),
```

Comment:
Either is fine, but a bit shorter like this:

```suggestion
            pbm.padding_border_sums.block +
                (content_size.block + clearance.unwrap_or_else(Length::zero)).into(),
```